lib/manifest.js

```diff
@@ -217,6 +217,15 @@ module.exports = [
             return internals.passthruOn('identity')(filename, value);
         }
     },
+    {   // Schwifty models
+        place: 'schwifty-models',
+        method: 'schwifty',
+        list: true,
+        useFilename: (filename, value) => {
+
+            return [].concat(value);
```

Comment:
What does this do?  Might not be necessary.  `server.schwifty(Model)` and `server.schwifty([Model])` should both work.